File: TeeNew/Right_Wall.pas
unit Right_Wall;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  {$ENDIF}
  Base, TeEngine, Series, TeeProcs, Chart;

type
  TRightWallForm = class(TBaseForm)
    Series1: TBarSeries;
    CheckBox1: TCheckBox;
    procedure CheckBox1Click(Sender: TObject);
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TRightWallForm.CheckBox1Click(Sender: TObject);
begin
  Chart1.RightWall.Visible:=CheckBox1.Checked
end;

procedure TRightWallForm.FormCreate(Sender: TObject);
begin
  inherited;
  Series1.FillSampleValues(6);
end;

initialization
  RegisterClass(TRightWallForm);
end.

File: TeeNew/Builder2010/Right_Wall.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Right_Wall.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Base"
#pragma resource "*.dfm"
TRightWallForm *RightWallForm;
//---------------------------------------------------------------------------
__fastcall TRightWallForm::TRightWallForm(TComponent* Owner)
        : TBaseForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TRightWallForm::CheckBox1Click(TObject *Sender)
{
  Chart1->RightWall->Visible = CheckBox1->Checked;
}
//---------------------------------------------------------------------------

void __fastcall TRightWallForm::FormCreate(TObject *Sender)
{
  Series1->FillSampleValues(6);
}
//---------------------------------------------------------------------------

